### PR TITLE
Clarify "usage" section of INSTALL file

### PR DESCRIPTION
**Pull request by otyshchenko1**
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -296,12 +296,22 @@ sudo tar -xf core-image-weston-MACHINE_NAME-xt.tar.bz2 -C /srv/domu/
 It worth mentioning that some changes to domain config files are necessary to switch between NFS
 and any of storage devices.
 
-For doing that:
-cd workspace_base/meta-xt-prod-gen3-test/recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/
+There are two options for doing that. Either by modifying source files followed by rebuilding
+the whole Domain-0's image or by modifying already installed files which implies unpacking an image
```
Comment (aanisov) [inline INSTALL]: ...the whole Domain-0' or by modifying domain config file of DomD right in a built initramfs image of Dom0, which....

File: INSTALL [hunk 2]
```diff
@@ -296,12 +296,22 @@ sudo tar -xf core-image-weston-MACHINE_NAME-xt.tar.bz2 -C /srv/domu/
...
+before and packing it back after (the "uirfs.sh" script which we will consider down the document
```
Comment (aanisov) [inline INSTALL]: ...an "uirfs.sh" script...

File: INSTALL [hunk 3]
```diff
@@ -296,12 +296,22 @@ sudo tar -xf core-image-weston-MACHINE_NAME-xt.tar.bz2 -C /srv/domu/
...
+is intended to help with that).
+
+In order to modify sources go to the files location in inner yocto:
+cd workspace_base/build/tmp/work/x86_64-xt-linux/dom0-image-thin-initramfs/1.0-r0/repo/meta-xt-prod-extra/recipes-extended/guest-addons/files
 
 - Edit domd-MACHINE_NAME.cfg to uncomment "extra" option which corresponds to NFS.
 - Edit domu-MACHINE_NAME.cfg to uncomment "extra" option which corresponds to NFS and comment "disk" option.
 
+and then rebuild Domain-0's image by hand:
```
Comment (aanisov) [inline INSTALL]: and then rebuild Domain-0 manually:

**Comment by otyshchenko1** [comment]
@iartemenko pls merge

**Pull request merged**